packages/flutter/test/foundation/reassemble_test.dart

```diff
@@ -2,16 +2,14 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart = 2.8
-
 @TestOn('!chrome')
 import 'dart:async';
 
 import 'package:flutter/foundation.dart';
 import '../flutter_test_alternative.dart';
 
 class TestFoundationFlutterBinding extends BindingBase {
-  bool wasLocked;
+  bool? wasLocked;
```

Comment:
Why not initialize this to "false"?

Reply:
I think making it a nullable ensures we detect any and all changes that `performReassemble` can deliver. Although it makes sense since our test only checks if it's changed to true, but a nullable type makes more sense in the general case.